scan/lib/scan/options.rb

```diff
@@ -515,7 +515,7 @@ def self.available_options
                                     default_value: false),
         FastlaneCore::ConfigItem.new(key: :number_of_retries,
                                     env_name: 'SCAN_NUMBER_OF_RETRIES',
-                                    description: "The number of times a test can fail before scan should stop retrying",
+                                    description: "In Xcode 13 and up, individual tests will retry on failure (see '-test-iterations' under `xcodebuild --help`). In Xcode 12 and under, the number of times a test can fail before scan should stop retrying",
```

Comment:
this documentation sounds better to me before the change to add 1, given it looks like it might be saying that number of retries is test iterations on Xcode 13. If we're adding 1 though, then I can think we can say that this is the "the number of times a test can fail" regardless of the Xcode version. I'd still lean towards the approach of adding 1 given the name "number of retries" sounds unambiguous and self-explanatory that it's the number of times a test can be retried. What do you think?

Reply:
another (possibly parallel) possibility that occurs to me is to deprecate this config and add a new one borrowing from Xcode the term "iterations" so fastlane is more aligned with the native tool naming, and older versions would be the special case to handle (by retrying iterations-1 times)

Reply:
I think you are right! This is way too much 😅 We'll go with "Number of time a test can fail". Users don't need to _really_ know about the inner workings 🤷‍♂️